explorer/client/package.json

```diff
@@ -91,6 +88,6 @@
     "rimraf": "^3.0.0",
     "ts-jest": "^24.1.0",
     "typescript": "^3.7.4",
-    "webpack": "4.41.5"
+    "webpack": "4.41.2"
```

Comment:
Do we need to upgrade this because of a conflict?

Reply:
I had to downgrade because `CRA` in feeds-ui complained with the [hard version lock](https://github.com/facebook/create-react-app/blob/v3.3.0/packages/react-scripts/package.json#L79). It doesn't use `craco`.

Reply:
Can we use `craco` and change the `webapack` version before merge?

Reply:
I thought this was related to `craco` but I was incorrect. The explorer is skipping the [preflight check](https://github.com/smartcontractkit/chainlink/blob/develop/explorer/client/.env) of CRA. It looks like `react-scripts` uses `webpack 4.41.5` on `master` but `4.41.2` on `react-scripts 3.3.0`. I'm going to leave this downgrade and remove the skip preflight check.

Reply:
Sure sounds good, let's deal with upgrading it at a later point